packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -723,25 +744,34 @@ async function handler(
 
   const bookingLocation = getLocationValueForDB(locationBodyString, eventType.locations);
   const customInputs = getCustomInputsResponses(reqBody, eventType.customInputs);
-  const teamMemberPromises =
-    users.length > 1
-      ? users.slice(1).map(async function (user) {
-          return {
-            email: user.email || "",
-            name: user.name || "",
-            timeZone: user.timeZone,
-            language: {
-              translate: await getTranslation(user.locale ?? "en", "common"),
-              locale: user.locale ?? "en",
-            },
-          };
-        })
-      : [];
+
+  const teamDestinationCalendar: DestinationCalendar[] = [];
```

Comment:
NIT: I think this should be `teamDestinationCalendars`. It'll make things easier to read and understand